### ISSUE TITLE
Option to group by site in the popup

### ISSUE THREAD

**Issue by fabianomichels** [olsh/Feedly-Notifier-Firefox#134]
Ability to group feeds by site name in the popup.
It would be nice if the popup would show the page name with a count of unread feeds. Then, when click on, show the list of feeds in the popup.

**Comment by olsh** [comment]
@fabianomichels  Thanks for the suggestion.
Have you tried to use `Show categories in popup` in the options? Maybe it's what you want.

**Comment by fabianomichels** [comment]
yes, I tried, but this only group by categories. My suggestion is to group like this:
![feeds](https://cloud.githubusercontent.com/assets/17577217/15511574/826f84a0-21b2-11e6-8c9e-8beed9c2f031.jpg)

**Comment by olsh** [comment]
I get the idea.
I'll implement it only after moving this addon to web extension and material design.
